rotkehlchen/tests/api/test_caching.py

```diff
@@ -152,7 +152,7 @@ def test_general_cache_refresh(rotkehlchen_api_server: 'APIServer'):
         ))
         assert_proper_response(response)
         assert patched_convex_query.call_count == 1, 'Convex pools should have been queried despite should_update_protocol_cache being False'  # noqa: E501
-        assert patched_curve_query.call_count == 1, 'Curve pools should have been queried despite should_update_protocol_cache being False'  # noqa: E501
+        assert patched_curve_query.call_count == 5, 'Curve pools should have been queried despite should_update_protocol_cache being False'  # noqa: E501
```

Comment:
hmm this test queries everything? Isn't it slow?

Reply:
it's fast because all the queries are mocked and rest is VCRed